.circleci/config.yml

```diff
@@ -453,7 +453,7 @@ jobs:
       - *setup_env
       - run:
           name: "Test via adhoc script"
-          command: ./yarn-project/boxes/run_tests blank-react
+          command: ./boxes/run_tests blank-react
```

Comment:
Can these be implemented using our standard container runner `cond_run_container` or `cond_spot_run_container`, or compose `cond_run_compose` or `cond_spot_run_compose` scripts?

Reply:
yes i think so!  I can do that in a follow up PR to keep this one digestible